src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -88,14 +107,28 @@ ExampleOperationalCredentialsIssuer::GenerateNodeOperationalCertificate(const Op
     ReturnErrorCodeIf(!noc.Alloc(kMaxCHIPDERCertLength), CHIP_ERROR_NO_MEMORY);
     uint32_t nocLen = 0;
 
-    ReturnErrorOnFailure(NewNodeOperationalX509Cert(request, CertificateIssuerLevel::kIssuerIsRootCA, pubkey, mIssuer, noc.Get(),
-                                                    kMaxCHIPDERCertLength, nocLen));
+    ReturnErrorOnFailure(NewNodeOperationalX509Cert(request, CertificateIssuerLevel::kIssuerIsIntermediateCA, pubkey,
```

Comment:
The `mIssuerId` in the `request` should be changed to `mIntermediateIssuer`. Otherwise, the NOC issuer will be set to the root.

Reply:
Thanks, updated.